lab_1_classify_by_unigrams/main.py

```diff
@@ -3,14 +3,21 @@
 Language detection
 """
 
-
 def tokenize(text: str) -> list[str] | None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation, digits and other symbols
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    new_text = ''
+    for i in text:
+        if i.isalpha():
+            new_text += i
+    tokens = list(new_text.lower())
```

Comment:
Consider making a list right away instead of making a string and then converting it to the list